grpc/oob/bmc/users_bmclibnext.go

```diff
@@ -25,7 +25,7 @@ func (b *bmclibNextUserManagement) Connect(ctx context.Context) error {
 	var errMsg repository.Error
 
 	client := bmclib.NewClient(b.host, "", b.user, b.password, bmclib.WithLogger(b.log))
-	client.Registry.Drivers = client.Registry.Using("redfish")
+	client.Registry.Drivers = client.Registry.FilterForCompatible(ctx)
```

Comment:
For my own reference, this is where that new method is defined.
https://github.com/jacobweinstock/registrar/blob/52643bf/registrar.go#L90-L123